ansible/playbooks/molecule/teleop_server_check_desktop_icons_docker/playbook.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Teleop deploy playbook for teleop server
+  import_playbook: ../../teleop-deploy.yml
+    reinstall=true
+    use_openvpn=false
+    ethercat_interface="enp0s25"
+    image="10.6.10.7:5000/dexterous-hand"
```

Comment:
hardcoded custom docker repository ?

Reply:
yes for pulling it from our nuc as our internet connection is very slow to pull it from docker

Reply:
I removed the 10.6 ip in my commit